test/powershell/Modules/Microsoft.PowerShell.Utility/Unblock-File.Tests.ps1

```diff
@@ -48,15 +40,15 @@ Describe "Unblock-File" -Tags "CI" {
 
     It "With '-Path': file exist" {
         Unblock-File -Path $testfilepath
-        Test-UnblockFile | Should -BeTrue
+        Test-UnblockFile
 
         # If a file is not blocked we silently return without an error.
         { Unblock-File -Path $testfilepath -ErrorAction Stop } | Should -Not -Throw
     }
 
     It "With '-LiteralPath': file exist" {
         Unblock-File -LiteralPath $testfilepath
-        Test-UnblockFile | Should -BeTrue
+        Test-UnblockFile
```

Comment:
Please revert the change - `Test-UnblockFile` should return True.

Reply:
Fixed

Reply:
Sorry for mistake. I changed Test-Unblock function so that currently it doesn't return true or false but just checks if the appropriate error was thrown.